src/mjolnir/speed_assigner.h

```diff
@@ -253,8 +254,10 @@ class SpeedAssigner {
                    const std::string& country_code,
                    const std::string& state_code) const {
 
-    // If we have config loaded we'll use that
-    if (!tables.empty() && FromConfig(directededge, density, country_code, state_code)) {
+    // See if we can get a valid speed loaded from configuration
+    auto configured_speed = FromConfig(directededge, density, country_code, state_code);
+    if (configured_speed != kUnconfiguredSpeed) {
+      directededge.set_speed(configured_speed);
```

Comment:
the main change here, other than storing an invalid value in the speed tables when `null` is encountered, is that we changed the signature of the `FromConfig` method. instead of setting the value on the edge and returning a boolean saying whether it did the assignment or not, it simply returns the speed it finds and the caller (this function) does with it what it wants. in this case it says, hey, was that a valid speed, if so i'll take it and move on with my life otherwise ill use the heuristic to figure out a speed to use for this edge